### PR TITLE
add notification when edit/delete device

### PR DESCRIPTION
**Pull request by mgold1234**
Fixes  # THEEDGE-3465

# Description
**this commit fix the missing notification in case of editing name of inventory and delete device,
when working with federation at the inventory side.**
Screenshot 2023-08-23 at 17 38 35

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Review by adeliaferreira — APPROVED** [review]
lgtm

File: src/Routes/Devices/Inventory.js [hunk 1]
```diff
@@ -155,17 +171,24 @@ const Inventory = ({
 
     let displayName = systemInstance.display_name;
     let removeSystems = [systemInstance.UUID];
+    const statusMessages = {
+      onSuccess: {
+        title: 'Delete operation finished',
+        description: `${displayName} has been successfully removed.`,
+      },
+      onError: { title: 'Error', description: 'Failed to update device name' },
```
[the remark this answers is not in the document]
Comment (acosferreira) [reply src/Routes/Devices/Inventory.js:179]: comment above, is the deletion message.

**Review by acosferreira — APPROVED** [review]
lgtm

**Review by ldjebran — APPROVED** [review]
lgtm

**Pull request merged**